sites/fast-website/src/app/components/index.ts

```diff
@@ -3,3 +3,5 @@ export { SiteSectionHeader } from "./section-header";
 export { FastFrame } from "./fast-frame";
 export { SiteNavigation } from "./navigation";
 export { SiteContentPlacement } from "./content-placement";
+export { FeatureCard } from "./feature-card";
+export { CardSection } from "./card-section";
```

Comment:
Could these have the prefix site added to them to align with the other components?

Reply:
Has that been settled?  I was following the discussion but don't recall seeing an official decision.  Brian was the last comment I saw on the matter.

Reply:
we need to prefix one way or another; we should use whichever prefix we've landed for the site

Reply:
Done